homeassistant/components/aws/notify.py

```diff
@@ -166,11 +167,13 @@ async def async_send_message(self, message="", **kwargs):
             _LOGGER.error("At least one target is required")
             return
 
-        message_attributes = {
-            k: {"StringValue": json.dumps(v), "DataType": "String"}
-            for k, v in kwargs.items()
-            if v is not None
-        }
+        message_attributes = {}
+        if kwargs.get(ATTR_DATA):
+            message_attributes = {
```

Comment:
The same pattern is seen for the AWSSQS service. Should that also be updated?

Reply:
Probably, but I didn't touch as I'm not able to test this service

Reply:
Ok.

Reply:
@cdce8p can you comment please ?

Reply:
I don't use the integration, so can't test it either. Sorry.